src/python/pants/engine/legacy/graph_test.py

```diff
@@ -16,7 +16,9 @@ def make_graph(name_to_deps: Dict[str, Tuple[str, ...]]) -> Dict[str, HydratedTa
   def make_ht(nm: str) -> HydratedTarget:
```

Comment:
The tests here don't properly store `HydratedTarget`s, so MyPy complains. For example, `address` is set as a `str` instead of a `BuildFileAddress`. That's fine here because it makes the test substantially easier to work with and the test ignores those fields anyways.

Reply:
Could we add that as a comment?

Reply:
Sure, I'll add it in the followup.